src/routes/admin/email-alias/[email]/+page.server.ts

```diff
@@ -0,0 +1,422 @@
+import apiNames from "$lib/utils/apiNames";
+import { authorize } from "$lib/utils/authorization";
+import type { PageServerLoad } from "./$types";
+import { message, setError, superValidate } from "sveltekit-superforms/server";
+import { error, fail } from "@sveltejs/kit";
+import { redirect } from "sveltekit-flash-message/server";
+import {
+  addPositionSchema,
+  addSpecialReceiverSchema,
+  addSpecialSenderSchema,
+  deleteEmailAliasSchema,
+  deleteSpecialReceiverSchema,
+  deleteSpecialSenderSchema,
+  removePositionSchema,
+  removeSpecialReceiverSchema,
+  removeSpecialSenderSchema,
+  setCanSendSchema,
+} from "./schema";
+import { isValidEmail } from "../emailutils";
+import keycloak from "$lib/utils/keycloak";
+import type { PrismaClient } from "@prisma/client";
+// eslint-disable-next-line no-restricted-imports -- the function should be in the API route
```

Comment:
I now added a special import route in `svelte.config.js` for the api route. I think this makes sense (and it solves the warning). It does not of course adrress what you brought up but IMO this could introduce unnecessary and complex logic

Reply:
Sounds good, as you pointed out many API routes are designed to be imported from many parts of the application (not all of them though, but perhaps that's a reasonable compromise)